lcls-twincat-motion/Library/POUs/Motion/PMPS/FB_PositionStatePMPSND_Core.TcPOU

```diff
@@ -72,15 +74,17 @@ ELSE
     eStatePMPSStatus := E_StatePMPSStatus.TRANSITION;
 END_IF
 
-fbMotionReadPMPSDB(
-    astPositionState:=astPositionStateMax,
-    fbFFHWO:=fbFFHWO,
-    sTransitionKey:=sTransitionKey,
-    sDeviceName:=sDeviceName,
-    bReadNow:=bReadDBNow,
-    astDbStateParams=>,
-    bError=>,
-);
+IF NOT bLocalPMPS THEN
```

Comment:
I expect this to work
I wonder if this should have a more straightforward name, e.g. `bUsePmpsDB`
Opinions welcome

Reply:
Your suggestions looks more consistent with the code base; updated.